data/brands/amenity/atm.json

```diff
@@ -17,6 +17,16 @@
         "operator:wikidata": "{source.tags.brand:wikidata}"
       }
     },
+    {
+      "displayName": "Hitachi Money Spot ATM",
+      "locationSet": {
+        "include": ["in"]
+      },
+      "tags": {
+        "amenity": "atm",
+        "brand": "Hitachi",
```

Comment:
https://github.com/osmlab/name-suggestion-index/wiki/Contributing#tags-required

NSI use wikidata as unified identifier for brand/network, so if it haven't been provided, the item won't be suggested in editor.